aten/src/ATen/native/cuda/UnaryComplexKernels.cu

```diff
@@ -24,12 +24,38 @@ __host__ __device__ static inline c10::complex<T> angle_wrapper(c10::complex<T>
   return std::arg(v);
 }
 
+const char angle_name[] = "angle_kernel";
 void angle_kernel_cuda(TensorIteratorBase& iter) {
-  AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES(iter.common_dtype(), "angle_cuda", [&]() {
+  auto dtype = iter.common_dtype();
+  if(isComplexType(dtype)) {
+#if AT_USE_JITERATOR()
+  static const auto angle_string = jiterator_stringify(
```

Comment:
Would be nice if this is indented. Currently it is under-indented.

Reply:
Updated! Thanks, Kshitij!